experimental/generation/composer-form-dialog.md

```diff
@@ -0,0 +1,61 @@
+# Form dialogs in Bot Framework Composer
+
+The form dialogs feature in Composer uses [dialog generation][generation] to automatically create a dialog from a form. The form dialog collects information for fields of the form while handling ambiguity, corrections, help, navigation and multiple fields at once. With form dialogs, Composer uses your form and creates the adaptive dialog, the bot responses, and the language model for you.
+
+**Note:** Form dialogs is a preview feature of Composer and should *not* be used in a production environment. To try an early version of form dialogs, follow the steps below.
+
+## Enabling the form dialogs preview
+
+To enable the form dialogs preview in Composer, follow these steps:
+
+1. Install the [Bot Framework Composer][link to Composer downloads].
```

Comment:
- Added a link to how to get composer running...
- Changed the reference to Bot Framework Composer (removed 'the' to make it more consistent with other Composer docs).

```suggestion
1. Install [Bot Framework Composer](https://docs.microsoft.com/en-us/composer/install-composer#build-composer-from-source).
```

Reply:
Users can either build Composer from the source or download the Electron app version. Change link to the article: [https://docs.microsoft.com/en-us/composer/install-composer](https://docs.microsoft.com/en-us/composer/install-composer) (no anchor link)?